src/component/update.modal.tsx

```diff
@@ -0,0 +1,136 @@
+"use client";
+import { useState, useEffect } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+import { toast } from "react-toastify";
+import { mutate } from "swr";
+
+interface IProps {
+  showModalUpdate: boolean;
+  setShowModalUpdate: (v: boolean) => void;
+  blog: IBlog | null;
+  setBlog: (value: IBlog | null) => void;
+}
+
+function UpdateModal(props: IProps) {
+  const { showModalUpdate, setShowModalUpdate, blog, setBlog } = props;
+
+  const [id, setId] = useState<number>(0);
+  const [title, setTitle] = useState<string>("");
+  const [author, setAuthor] = useState<string>("");
+  const [content, setContent] = useState<string>("");
+
+  useEffect(() => {
+    if (blog && blog.id) {
+      setId(blog.id);
+      setTitle(blog.title);
+      setAuthor(blog.author);
+      setContent(blog.content);
+    }
+  }, [blog]);
+  const handleSubmit = () => {
+    if (!content && !content && !author) {
+      toast.error("Not empty data !");
+      return;
+    }
```

Comment:
same as https://github.com/hoihmyit/crud_app/pull/1/files#r1398408327

Reply:
Thank you for your feedback and I have corrected it above